UberCore.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+
+  s.name         = "UberCore"
```

Comment:
Can we share Podspec data between the two binaries in the parent folder to minimize duplicate info that needs to be updated?

Reply:
Will look into this.

Reply:
Done.